src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -598,8 +598,7 @@ private function addDefaultsSection(ArrayNodeDefinition $rootNode): void
                 return $normalizedDefaults;
             });
 
-        [$publicProperties, $configurableAttributes] = ApiResource::getConfigMetadata();
-        foreach (array_merge($publicProperties, $configurableAttributes) as $attribute => $_) {
+        foreach (ApiResource::getConfigMetadata()[1] as $attribute => $_) {
```

Comment:
I don't think this change should be here.

Reply:
I just reverted the changes introduced in https://github.com/api-platform/core/commit/7d2cfbb3766b4fab4fd5789759b1321385c7a855, I haven't checked if something besides the indentation was changed.

Reply:
But it makes sense. I will do the change.

Reply:
allrighty, fixed.